python/cudf/udf_cpp/nrt/numba_cuda_runtime.cuh

```diff
@@ -0,0 +1,129 @@
+/*
+* SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+* SPDX-License-Identifier: Apache-2.0
```

Comment:
Closing the loop here since there's been some offline discussion. There's a pending question here about how to distribute this license in a compliant manner since it won't be present in the PTX this file compiles to. We decided to leave this as a TODO so that it won't block technical work for now, but we will need to resolve this before we merge this to dev and start distributing nightlies.